Podfile

```diff
@@ -23,10 +23,10 @@ workspace 'WordPress.xcworkspace'
 ## ===================================
 ##
 def wordpress_shared
-  pod 'WordPressShared', '~> 2.3'
```

Comment:
I looked at the last couple of times this reference, which was in https://github.com/wordpress-mobile/WordPress-iOS/pull/22376 and https://github.com/wordpress-mobile/WordPress-iOS/pull/20511. From those PRs, I see only the commit hash was updated in the main PR, then apps infra later created a release tag in https://github.com/wordpress-mobile/WordPress-iOS/commit/7fe39470a3d290694cd30a9ec7b29986eae18739 and https://github.com/wordpress-mobile/WordPress-iOS/commit/d2889441e7534980b00b007fc6d649655de87639.

@mokagio, is this approach the right one? I've followed that lead with this PR, but would be keen to learn if I should have done something differently here. Thank you!